.github/workflows/unittests.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         export PATH=${PATH}:${GOPATH}/bin
         GO111MODULE=off go get -u github.com/gordonklaus/ineffassign
         GO111MODULE=off go get -u golang.org/x/lint/golint
-        make check-fmt lint test
+        make lint vet test
```

Comment:
no more ./contrib/scripts/check-fmt.sh?

Reply:
It's a `lint` dep so it was effectively run twice. I think just once is enough :)

Reply:
maybe it'll catch something the second time :)